master/buildbot/test/unit/test_steps_transfer.py

```diff
@@ -866,8 +866,8 @@ def testBasic(self):
                 self.assertEquals(
                     data, json.dumps(dict(sourcestamps=[ss.asDict()], properties={'key1': 'value1'})))
                 break
-        else:
-            self.assert_(False, "No downloadFile command found")
+            else:
+                self.assertRaise(ValueError, "No downloadFile command found")
```

Comment:
Is there `self.assertRaise` in `TestCase`?

I know about `self.assertRaises`, and in this place I think original code is more corresponds to `raise ValueError("No downloadFile command found")/`